src/client/puppet-service.ts

```diff
@@ -353,7 +353,7 @@ class PuppetService extends PUPPET.Puppet {
 
       default:
         // Huan(202003): in default, the `type` type should be `never`, please check.
-        throw new Error('eventType ' + type + ' unsupported! (code should not reach here)')
+        log.error('eventType ' + type + ' unsupported! (code should not reach here)')
```

Comment:
Please link to the related issue so that this warning message can guide the end user/developer to the right place for more information

Reply:
OK